src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java

```diff
@@ -39,6 +41,7 @@ public final class MediaHelper {
 	private static final Logger LOGGER = LoggerFactory.getLogger(MediaHelper.class);
 
 	private static final Gson GSON = new Gson();
+	static final ProcessLocator FFMPEG_LOCATOR = new PathLocator();
```

Comment:
Maybe rename to PROCESS_LOCATOR?

Reply:
To me it's more readable explicitly calling it "ffmpeg"